recipes/spotiflow/meta.yaml

```diff
@@ -0,0 +1,81 @@
+{% set name = "spotiflow" %}
+{% set version = "0.3.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/spotiflow-{{ version }}.tar.gz
+  sha256: d3faf71ab15f3eef65404489983ce48a4bc094212896ea97274bf970dafc2eb8
+
+build:
+  skip: True  # [win or py<39 or py>=312]
+  entry_points:
+    - spotiflow-predict = spotiflow.cli.predict:main
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - numpy                                  # [build_platform != target_platform]
+    - {{ compiler('c') }}
+    - {{ stdlib("c") }}
+    - {{ compiler('cxx') }}
+    - make
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+  host:
+    - python
+    - setuptools >=40.8.0
+    - setuptools-scm >=6.2
+    - wheel
+    - numpy
+    - pip
+    - {{ compiler('c') }}
+    - {{ stdlib("c") }}
+    - {{ compiler('cxx') }}
+    - make
```

Comment:
I only meant the OpenMP libraries. They currently have a bug where they need to be in build and host, all other should only be in build.
```suggestion
```